doc/lua_api.md

```diff
@@ -75,40 +77,52 @@ All integer values have a range with a defined minimum and maximum.
 Integer ranges are written as [min, max] and are inclusive. E.g. the
 integer range [0, 255] contains all integers from 0 to 255,
 *including* 0 and 255.
-When the documentation does not specify a range or other restrictions,
-assume a range of [-2^37, 2^37].
 
 Some ranges in this documentation like [-2^15, 2^15-1] occur frequently,
 and may be abbreviated like so:
 
-    [int16]  = [-2^15, 2^15-1] = [-32768, 32767]
-    [int32]  = [-2^31, 2^31-1] = [-2147483648, 2147483647]
-    [uint16] = [0,     2^16-1] = [0, 65535]
-    [uint32] = [0,     2^32-1] = [0, 4294967295]
-    [uint64] = [0,     2^64-1] = [0, 18446744073709551615]
+    [s16]  = [-2^15, 2^15-1] = [-32768, 32767]
+    [s32]  = [-2^31, 2^31-1] = [-2147483648, 2147483647]
+    [slua] = [-2^53, 2^53]   = [-9007199254740992, 9007199254740992]
+    [u16]  = [0,     2^16-1] = [0, 65535]
+    [u32]  = [0,     2^32-1] = [0, 4294967295]
+    [u64]  = [0,     2^64-1] = [0, 18446744073709551615]
+    [ulua] = [0,     2^53]   = [0, 9007199254740992]
+    [imagesize]              = [1, 23000]
 
-(uint = "unsigned integer", a term borrowed from C++ for integer types
-that cannot store negative integers)
+(s = "signed", ranges that include negative integers;
+ u = "unsigned", ranges that don't include negative integers;
+ these terms are borrowed from the C language)
```

Comment:
very much a nitpick: the C reference confuses me a bit. it's `uint<x>_t` and `int<x>_t` in `stdint.h`.

`s` and `u` are just common abbreviations that tend to get used, in our case they happen to be defined by `irrTypes.h`.

Reply:
Read again: The TERMS "signed" and "unsigned" are borrowed from C, not the abbreviations u16, u32, etc.